pkg/resource/resource.go

```diff
@@ -102,12 +38,20 @@ func (a *AbstractResource) TerraformType() string {
 	return a.Type
 }
 
-func (a *AbstractResource) CtyValue() *cty.Value {
-	return nil
+func (a *AbstractResource) Attributes() *Attributes {
+	return a.Attrs
+}
+
+func (a *AbstractResource) HumanReadableAttributes() map[string]string {
```

Comment:
I think this logic should not be related to the resource struct.
 
We put too many thinks in resource : the schema, the data itself, the human readable function that is bound to the schema 🤔 not very clear IMHO we should rethink this.

Reply:
yeah I think this is the same discussion as for separation between the schema and the resource. It is handy that abstractresource just implement the HumanReadableAttributes interface I think

Reply:
For me this is more related to the `View` than the `Model` part if we think MVC. I had in mind that the resource is our model and is only a data container object. This should not be bound to a resource.